gensim/corpora/_mmreader.c

```diff
@@ -0,0 +1,10902 @@
+/* Generated by Cython 0.29.14 */
```

Comment:
This and other `.c` files created by Cython compilation should *not* be added to version control. (In fact, if we think no hand-authored `.c` files will be part of the project, a patern excluding them could be added to `.gitignore`.)

Reply:
Should I also add header files(.h) in gitignore?

Reply:
No, some header files are real, like `wrap_stdint.h`.